openvalidators/config.py

```diff
@@ -166,6 +166,14 @@ def add_args(cls, parser):
         default=4096,
     )
 
+    parser.add_argument(
+        "--neuron.axon_off",
```

Comment:
A comment in the code here would be useful to explain why the axon should be on, and warn that turning it off will e.g. risk being blacklisted.